src/components/AdminSettings.vue

```diff
@@ -429,7 +429,7 @@ export default {
 		},
 		checkIfDemoServerIsActive() {
 			this.settings.demoUrl = this.demoServers ? this.demoServers.find((server) => server.demo_url === this.settings.wopi_url) : null
-			this.settings.CODEUrl = this.CODEInstalled ? window.location.protocol + '//' + window.location.host + OC.generateUrl('/apps/richdocumentscode/proxy.php?req=') : null
+			this.settings.CODEUrl = this.CODEInstalled ? window.location.protocol + '//' + window.location.host + OC.getRootPath() + '/apps/richdocumentscode/proxy.php?req=' : null
```

Comment:
If that is supposed to link to the actual proxy.php file now, please use something like `OC.filePath('richdocuments', 'ajax', '') + 'proxy.php'`. The app might be located in a different directory than apps/ as you can have different app root in Nextcloud.

Reply:
Done. Thank you! :)